tests/test_get_signature.py

```diff
@@ -14,7 +16,10 @@ def some_function(*args, **kwargs):
 
 
 class Dog:
-    def woof(self, volume, good_boy=True, name='doggy'):
+    def woof(self, volume, good_boy: bool = True, name: str = 'doggy') -> int:
+        pass
+
+    def hard_method(self, arg: Union[int, Tuple[int, int]], arg2: int = 0) -> int:
         pass
```

Comment:
Could we have here a test with the expected type hint? `Union[int, Tuple[int, int]]` should be in the final docstring.

Reply:
Added a test in 8a38a3e.